client/components/common/tce-core/AddElement/index.vue

```diff
@@ -1,21 +1,23 @@
 <template>
   <div class="add-element-container">
-    <v-btn
-      v-if="large"
-      @click.stop="isVisible = true"
-      color="blue-grey darken-3"
-      text
-      class="mt-3 mb-4">
-      <v-icon class="pr-2">{{ icon }}</v-icon>{{ label }}
-    </v-btn>
-    <v-btn
-      v-else
-      @click.stop="isVisible = true"
-      icon
-      text
-      color="blue-grey darken-3">
-      <v-icon>{{ icon }}</v-icon>
-    </v-btn>
+    <slot :addElement="showElementPicker">
```

Comment:
Why did you need to add a slot? / what did you gained with method extraction?

Reply:
The goal was to make the `Add question element` button stylistically identical to the `Add answer` button (SC, MC, FB, NR...) and position it correctly where needed. 

`tce-core/add-element` is most likely reused within some custom containers and/or elements so I didn't want to change it. Adding a slot with fallback content seemed better choice than styling it with v-deep from `tce-core/question-container/question`.